clients/githubrepo/client.go

```diff
@@ -237,3 +238,43 @@ func CreateOssFuzzRepoClient(ctx context.Context, logger *log.Logger) (clients.R
 	}
 	return ossFuzzRepoClient, nil
 }
+
+// GetClients returns a list of clients for running scorecard checks.
+// TODO(repo): Pass a `http.RoundTripper` here.
+func GetClients(ctx context.Context, repoURI, localURI string, logger *log.Logger) (
```

Comment:
This should probably belong in `checker` package maybe since it looks more like a factory fn which can return either GitHub client or localdir client.